composer.json

```diff
@@ -53,6 +53,7 @@
         "payum/payum-bundle": "~1.0",
         "sensio/distribution-bundle": "~2.3",
         "stof/doctrine-extensions-bundle": "~1.1",
+        "gedmo/doctrine-extensions": "dev-master as 2.4.11",
```

Comment:
Do we need this after removing [NestedTreeRepositoryTrait](https://github.com/Sylius/Sylius/pull/3773/files#r50019392)?